src/hope/apps/changelog/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class ChangelogConfig(AppConfig):
     name = "hope.apps.changelog"
+
+    def ready(self) -> None:
+        import hope.models  # noqa
```

Comment:
Why do we have to import ALL models in each apps.py ready?

Reply:
checking....